bot/client.py

```diff
@@ -127,25 +127,40 @@ async def cog_command_error(self, context, error):
         await context.send('An error occurred: {}'.format(str(error)))
         logger.error('An error occurred: {}'.format(str(error)))
 
-    @commands.command(invoke_without_subcommand=True)
+    @commands.hybrid_command(
+        name='join',
+        invoke_without_subcommand=True,
+        with_app_command=True,
+        description='Make Myinstants bot join.',
+    )
     async def join(self, context):
         if not context.author.voice:
             raise VoiceError('You are not connected to a voice channel.')
 
+        await context.send('Joining user\'s channel.')
```

Comment:
That's a breaking change. You can't answer commands twice
![image](https://user-images.githubusercontent.com/3418974/229374464-25f2a535-ca5e-4583-8e25-1dfb0e7dfb31.png)

Reply:
Removed this feedback and changed how `join` function actually works. IMO it doesn't need to be an action. Nobody would want the bot to join without any purpose. Also, leaving it without any feedback to the user is not a good UX!